index.html

```diff
@@ -73,8 +73,8 @@
             <div class="main-image code-snippet">
          
                 <iframe
-                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=solarized+dark&wt=bw&l=htmlmixed&width=690&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=true&fl=1&fm=IBM+Plex+Mono&fs=12px&lh=143%25&si=false&es=2x&wm=false&code=%253C%21--Copy%2520the%2520below%2520line%2520of%2520code%2520and%2520paste%2520it%2520in%2520the%2520head%2520tag%2520of%2520your%2520HTML%2520file%2520--%253E%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Furltobeentered.netlify.app%252Fcomponents%252Fstyle.css%2522%253E"
-                style="width: 857px; height: 259px; border:0; transform: scale(1); overflow:hidden;"
+                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=solarized+dark&wt=bw&l=htmlmixed&width=888&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=IBM+Plex+Mono&fs=12px&lh=143%25&si=false&es=2x&wm=false&code=%253C%21--Copy%2520the%2520below%2520line%2520of%2520code%2520and%2520paste%2520it%2520in%2520the%2520head%2520tag%2520of%2520your%2520HTML%2520file%2520--%253E%250A%253Clink%2520rel%253D%2522stylesheet%2522%2520href%253D%2522https%253A%252F%252Furltobeentered.netlify.app%252Fcomponents%252Fstyle.css%2522%253E"
+                style="width: 692px; height: 112px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
try to avoid inline styles as much as possible. move them into a separate class and reuse them throughout it wont affect your iframe.